client/packages/system/src/Stock/api/hooks/keys.ts

```diff
@@ -0,0 +1 @@
+export const STOCK = 'stock';
```

Comment:
nice nice, although I would make it more specific. `STOCK_LINE`

Reply:
Although maybe it doesn't make it more specific, and the point of the constants is being able to navigate to references, which would explain the context any ways